web/webpack.assets.json

```diff
@@ -50,6 +50,11 @@
         "./src/portico/tabbed-instructions"
     ],
     "landing-page": [
+        "./src/bundles/portico",
```

Comment:
This diff is a little obfuscated; `"landing-page"` is the entry point on most landing pages; this removes the inclusion of `./styles/portico/pricing_plans` for those, and duplicates the previous stack of assets on the more explicit `"landing-page-with-pricing"` entry point.